addons/portal/data/mail_templates.xml

```diff
@@ -6,7 +6,9 @@
             <p>Dear <span t-esc="partner.name"/>,</p>
             <p>You have been invited to access the following document:</p>
             <br/>
-            <a t-attf-href="#{share_link}" style="background-color: #875A7B; padding: 10px; text-decoration: none; color: #fff; border-radius: 5px; font-size: 12px;"><strong>Open </strong><strong t-esc="record.display_name"/></a><br/>
+            <div class="mt-2">
```

Comment:
-> Thinking about it, I don't think the change works anyway because this will be sent as an email where bootstrap will not work.

Would just either insert another `<br>` or add margin-top as an inline `<... style="margin-top: 1rem">`

Reply:
@thiryr 
these branch is moved to master branch
Community PR : https://github.com/odoo/odoo/pull/126645
Thank you